docs/auth/add-auth-provider.md

```diff
@@ -123,9 +123,9 @@ due to its comprehensive set of supported authentication
 
 [1.](#create-new-auth-provider-module) Create a new auth provider module
 
-[3.](#adding-an-oauth-based-provider) or [adding a proxy auth based provider](#creating-proxy-auth-based-provider) depending on your needs.
+[2.](#adding-an-oauth-based-provider) or [adding a proxy auth based provider](#creating-proxy-auth-based-provider) depending on your needs.
 
-[4.](#add-the-provider-to-the-backend) Add the provider to the backend.
+[3.](#add-the-provider-to-the-backend) Add the provider to the backend.
```

Comment:
I think this should actually be something like:

```
[1.](#create-new-auth-provider-module) Create a new auth provider module or [add a proxy auth based provider](#creating-proxy-auth-based-provider) depending on your needs.
[2.](#add-the-provider-to-the-backend) Add the provider to the backend.
```

I think this doc needs some more rewording but this makes the most logical sense IMHO for now at least 😅